### DISCUSSION TITLE
Can't connect my database to Data Studio

### DISCUSSION THREAD

**Discussion by hkajdan** [supabase/supabase#6531] [Questions] [answered]
Hello,

I have an issue on my new project, seems like I can't connect it to Data Studio through the DS Postgresql connector.

I've tried many ways but to no avail. My other project works perfectly and I tried to replicate the same settings, still doesn't work. I've tested other ways to connect to the db, they all work, everything but Data Studio.

Nothing in particular in the logs, I starting to get desperate, so if anybody has an idea, I'm open !

![image](https://user-images.githubusercontent.com/67869559/164766356-503cde31-1805-40af-9f27-dba398fcbab1.png)

Thank you !

**Comment by burggraf** [comment, marked as answer]
At this time, Data Studio only supports up to PostgreSQL version 12.  If you need support for PostgreSQL 14, maybe contact Google regarding a newer database connector.

See: https://support.google.com/datastudio/answer/7288010?hl=en#zippy=%2Cin-this-article

**Reply by eelco** [reply]
I’m also getting an error, but Google now claims to support up to 14.

**Comment by hkajdan** [comment]
Hello Mark,

Thank you for your answer, at least I can stop tearing my hair over this ! 
I'll try to contact Google, not holding my breath though...

Have a nice day !
